dev/integration_tests/ui/lib/keyboard_resize.dart

```diff
@@ -60,7 +60,7 @@ class _MyHomePageState extends State<MyHomePage> {
       ),
       floatingActionButton: FloatingActionButton(
         key: const Key(keys.kUnfocusButton),
-        onPressed: () { textField.focusNode.unfocus(); },
+        onPressed: () { textField.focusNode?.unfocus(); },
```

Comment:
I think since we're assigning a new focus node each time, this can be a `!`.

The whole implementation is incorrect, however: focus nodes should not be created with each build, or the focus gets reset with each build. The focus node should be a member of the state object and passed in, then this line wouldn't need to reference a local variable (it would just reference the state's focus node), and the `TextField` could be created inline.